### ISSUE TITLE
Add REV mode to the KAP140

### ISSUE THREAD

**Issue by wlbragg** [c172p-team/c172p#1344]
There is a fix by @hbeni to the KAP140 on the c182s/t. All that is needed is to test and merge the newly created PR.
It needs to be tested by someone familiar with KAP140 operations before merging.

**Comment by hbeni** [comment]
> All that is needed is to test

Did it and documented it at the PR: looks good for me

**Comment by bitwisetech** [comment]
I sourced a ticket regarding variations in the static pressure signal when the mouse LMB is pressed off-window.  The c172p seems sensitive to variations in that signal which is differentiated to yield ROC.   Too, the elevator PID seems to be doing a double integration using elevator to respond to target ROC.  
  I've posted a modified KAp140.xml at: www.bitwisetech.com/fgfs/c172p/Systems/KAP140.xml: 
the elevator drive is split into two PIDs with target pitch as the intermediate signal.  Gating for signal injection ( keyboard hotkeys )  for each stage is left in place, I'm looking at  wing droop at lower airspeeds.  The filter for derived ROC  has and increased time constant. 
Season's best wishes.
